packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -243,7 +269,13 @@ export async function updateRevision(
 ) {
   // If editing defaultValue or rules, require the revision to be a draft
   if ("defaultValue" in changes || changes.rules) {
-    if (revision.status !== "draft") {
+    if (
+      !(
+        revision.status === "draft" ||
+        revision.status === "pending-review" ||
+        revision.status === "reviewed"
```

Comment:
Same here with the status.  I don't think `reviewed` is one of the statuses - need to change to `approved` and/or `changes-requested`, right?